lib/bundle/installer.rb

```diff
@@ -26,16 +27,21 @@ def install(entries)
           arg << entry.options
           Bundle::TapInstaller
         end
-        case cls.install(*arg)
-        when :success
-          puts Formatter.success("#{verb} #{entry.name}")
-          success += 1
-        when :skipped
-          puts "Using #{entry.name}"
-          success += 1
-        else
-          puts Formatter.error("#{verb} #{entry.name} has failed!")
-          failure += 1
+        begin
```

Comment:
Could this perhaps be split into a separate method to encapsulate the exception handling in there?

Reply:
I think so. I'm thinking I'm going to refactor the whole big method.